src/containers/guest-home-page/Welcome.js

```diff
@@ -13,18 +13,14 @@ const scrollTo = routes.guestNavBar.whatCanYouDo.route
 
 const Welcome = () => {
   const { t } = useTranslation()
-  const size = useBreakpoints()
+  const { isDesktop, isTablet, isMobile } = useBreakpoints()
 
-  const images = {
-    desktop: titleMd,
-    tablet: titleSm,
-    mobile: titleXs
-  }
+  const images = (isDesktop && titleMd) || (isTablet && titleSm) || (isMobile && titleXs)
```

Comment:
this is not images, here we assign one img

Reply:
done.